docs/oidc.md

```diff
@@ -20,6 +20,10 @@ oidc:
   # Specified/generated by your OIDC provider
   client_id: "your-oidc-client-id"
   client_secret: "your-oidc-client-secret"
+  # alternatively, set `client_secret_path` to read the secret from the file.
```

Comment:
I want to discuss: Can we just make the "old" option understand paths?

It would be nice not having to deal with having mutual exclusive options and having to deal with errors and user support for those. 

If the old entry is changed to accept a path, and then it can first check if it is a path, and then fall back to the secret?

Reply:
I understand the sentiment, but I can't figure out how to do this.

How can we determine if something is a path? A few ideas:
- "If it looks like a path, it's a path", we may get false positives. Is "supersecret" a path? How about `/&!!.`?
- If we `stat` and the file exists? Sounds good. But if the file does not exist due to a configuration error, we treat the string as a password. But understanding that headscale uses an incorrect OIDC password is very non-trivial.

Reply:
I think this is fair. Lets keep it as a separate option for now.